app/views/govuk_publishing_components/components/docs/summary_list.yml

```diff
@@ -73,13 +72,10 @@ examples:
       The text of the link can be customised using the `link_text` option. Take care that the provided `link_text` still makes sense to screen readers when combined with the title. For instance, `Reorder` link text and `Items` title becomes `Reorder Items`, which reads fine, but link text of `Summary` would read as `Summary Items`, which does not make sense.
```

Comment:
Can't put a comment on L68 so I'll just leave it here.

Would this variant be better renamed to `with_no_hidden_text`, or something like that? 

Or alternatively could the example be adjusted to better represent what is detailed in the description? The example showcases what the `link_text_no_enhance` parameter does, but doesn't show how to replace the default visually hidden text with your custom one.

Reply:
I think this is a bit out of scope for this PR but it seems important so I'm adding it in. I think clarifying the description is better than renaming the example, so I've given it a shot. Hopefully this is better.